backend/scripts/diagnose_firecrawl.py

```
@@ -0,0 +1,268 @@
+用于测试 Firecrawl API 连接、代理配置和网络问题
+"""
+
+import asyncio
+import os
+import sys
+from pathlib import Path
+
+# 添加项目根目录到Python路径
+sys.path.insert(0, str(Path(__file__).parent.parent))
+
+import requests
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)